appveyor.yml

```diff
@@ -35,11 +35,12 @@ before_deploy:
 
 artifacts:
   - path: OpenRA-%APPVEYOR_REPO_TAG_NAME%.exe

```

Comment:
You mean http://www.appveyor.com/docs/packaging-artifacts#pushing-artifacts-from-scripts? We are not pushing from a PowerShell script.